Slapper/src/slapper/entities/SlapperHuman.php

```diff
@@ -37,7 +37,7 @@ public function spawnTo(Player $player)
 
             $add = new PlayerListPacket();
             $add->type = 0;
-            $add->entries[] = [$uuid, $entityId, isset($this->namedtag->MenuName) ? $this->namedtag["MenuName"] : "", $this->skinId, $this->skin];
+            $add->entries[] = [$uuid, $entityId, isset($this->namedtag->MenuName) ? $this->namedtag["MenuName"] : "", $this->setNameTagVisible(), $this->setNameTagAlwaysVisible(), $this->skinId, $this->skin];
             $player->dataPacket($add);
```

Comment:
This is the wrong place to put such a thing, you're more likely to cause client crashes.